skills_asses/src/app/login/login.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl } from "@angular/forms";
+import { Validators } from "@angular/forms";
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'login',
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.css'
+})
+export class LoginComponent {
```

Comment:
**add this get**
 `get showErrorMessage(): boolean {`
  `  return this.loginForm.controls.username.errors?.['required']`
     ` || this.loginForm.controls.password.errors?.['required'] || this.loginForm.controls.password.errors['minlength']`
  `}`